CONTRIBUTING.md

```diff
@@ -1 +1,7 @@
-Please see [Contribution Guidelines](https://developers.mattermost.com/contribute/getting-started/)
+# "Your contribution will help the world💖"
+
+## [Contribute to Mattermost](https://mattermost.com/contribute/)
```

Comment:
The page you linked contains information about how to contribute to Mattermost in general. Could you please move it after the link to https://mattermost.com/contribute/ and make more clear, that it's about other projects at Mattermost?

Reply:
Ok